app/templates/partials/summary/summary.html

```diff
@@ -1,53 +1,59 @@
-      {% endif %}
-      <div class="summary__block">
-
-        {% for block in group.blocks %}
-
-          <div class="summary__items">
-
-            {% for question in block.questions %}
-
-              <div class="summary__question" id="{{question.id}}" data-qa="summary-question-title">
-                {{question.number or ''}} {{question.title|striptags}}
-              </div>
-
-              {% for answer in question.answers %}
-                {% if question.answers|length > 1 and answer.label %}
-                  <div class="summary__question summary__question--sub">
-                    {{answer.label|striptags}}
-                  </div>
-                {% endif %}
-
-                <div class="summary__answer">
-                  <div class="summary__answer-text" id="{{answer.id}}-answer" data-qa="{{answer.id}}-answer">
-                    {%- if answer.value is none or answer.value == '' -%}
-                      {{not_answered}}
-                    {%- else -%}
-                      {% include theme(['partials/summary/' ~ answer.type ~ '.html', 'partials/summary/default.html']) %}
-                     {%- endif -%}
-                   </div>
-
-                  {% if content.summary.answers_are_editable %}
-                    <div class="summary__edit">
-                      <a href="{{ block.link }}#{{answer.id}}" class="summary__edit-link" aria-describedby="{{answer.id}} {{answer.id}}-answer" data-qa="{{answer.id}}-edit" {{helpers.track('click', 'Summary', 'Edit click')}}>Edit <span class="u-vh">your answer</span></a>
-                    </div>
-                  {% endif %}
-                </div>
-              {% endfor %}
-            {% endfor %}
+<div class="summary">
+	{% for group in content.summary.groups %}
+        {% if group.title %}
+            <h2 class="summary__title neptune" id="{{ group.id }}">{{ group.title }}</h2>
+        {% endif %}
+
+        <div class="summary__block">
+            {% for block in group.blocks %}
+                {% for question in block.questions %}
+
+                    {% set has_textarea = has_answer_type(question, 'textarea') %}
+                    {% set has_multiple_answers = question.answers|length > 1 %}
+
+                    <div class="summary__item {{ 'summary__item--multiple' if has_multiple_answers }}">
+                        <div class="grid">
+                            <div class="grid__col {{ 'col-12@m' if has_multiple_answers else 'col-12@xs' }} {{ 'col-10@m' if has_textarea else 'col-6@m' }}">
+                                <div class="summary__question mars" id="{{ question.id }}" data-qa="summary-question-title">{{ question.number or ''}} {{question.title|striptags }}</div>
+                            </div>
+                    {{ '</div>' if has_multiple_answers }}
+
+                    {% for answer in question.answers %}
+
+                        {% set is_textarea = has_answer_type(answer, 'textarea') %}
```

Comment:
this is confusing, I'm guessing your just trying to get the answer_type retuned when it matches 'textarea'. In which case, should update the Jinja filter input to say `question_or_answer`, and add a docstring to explain

Reply:
Will separate this into 2 functions to make it clear what it is doing.